tools/clang/unittests/HLSL/OptionsTest.cpp

```diff
@@ -198,7 +198,7 @@ TEST_F(OptionsTest, ReadOptionsWhenInvalidThenFail) {
   const wchar_t *ArgsNoArg[] = {L"exe.exe", L"hlsl.hlsl", L"/E", L"main",
                                 L"/T"};
   const wchar_t *ArgsUnknown[] = { L"exe.exe", L"hlsl.hlsl", L"/E", L"main",
-    L"/T" L"ps_6_0", L"--unknown"};
+    (L"/T" L"ps_6_0"), L"--unknown"};
```

Comment:
Looks like your other change got mixed in here. Might want to rebase to avoid confusion

Reply:
Should be fixed now :)